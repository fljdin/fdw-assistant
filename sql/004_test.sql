
INSERT INTO config (source, target, pkey, condition, batchsize, trunc) VALUES
    ('source.withkeywords', 'public.withkeywords', 'id', null, null, true);

SELECT target, statement FROM plan('{public.withkeywords}');

-- start(11) should succeed and "limit" must be quoted
CALL start(11);
