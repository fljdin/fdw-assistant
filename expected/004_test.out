INSERT INTO config (source, target, pkey, condition, batchsize, trunc) VALUES
    ('source.withkeywords', 'public.withkeywords', 'id', null, null, true);
INSERT 0 1
SELECT target, invocation FROM plan('{public.withkeywords}');
       target        |       invocation        
---------------------+-------------------------
 public.withkeywords | CALL assistant.copy(7);
(1 row)

-- copy(7) should succeed and "limit" must be quoted
CALL copy(7);
NOTICE:  Executing: TRUNCATE public.withkeywords
NOTICE:  Executing: SELECT count(id) FROM source.withkeywords WHERE id > 0 
NOTICE:  Executing: INSERT INTO public.withkeywords SELECT id, "limit" FROM source.withkeywords WHERE id > 0  ORDER BY id 
CALL
